exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporter.java

```diff
@@ -73,24 +71,15 @@ public CompletableResultCode export(T exportRequest, int numItems) {
 
           exporterMetrics.addFailed(numItems);
 
-          byte[] body = null;
-          try {
-            body = httpResponse.responseBody();
```

Comment:
The fact that this `responseBody()` throws when called multiple times is a bug. There's nothing in the contract of `HttpSender.Response.responseBody()` that suggests that you can't call it multiple times. While this avoids the issue, I think we need to fix the root of the problem in `OkHttpHttpSender` with something like:
```
onResponse.accept(
    new Response() {
      @Nullable
      private byte[] bodyBytes;
      @Override
      public int statusCode() {
        return response.code();
      }

      @Override
      public String statusMessage() {
        return response.message();
      }

      @Override
      public byte[] responseBody() throws IOException {
        if (bodyBytes == null) {
          bodyBytes = body.bytes();
        }
        return bodyBytes;
      }
    });
```

Reply:
Alright, I've reverted that last change and have added this.